components/emotionalRegulation/E.tsx

```diff
@@ -0,0 +1,190 @@
+  const { active, data, deleteTask } = props;
+
+  const activeAnim = useRef(new Animated.Value(0));
+  const style = useMemo(
+    () => ({
+      transform: [
+        {
+          scale: activeAnim.current.interpolate({
+            inputRange: [0, 1],
+            outputRange: [1, 1.01],
+          }),
+        },
+      ],
+    }),
+    []
+  );
+
+  useEffect(() => {
+    Animated.timing(activeAnim.current, {
+      duration: 300,
+      easing: Easing.bounce,
+      toValue: Number(active),
+      useNativeDriver: true,
+    }).start();
+  }, [active]);
+
+  return (
+    <Animated.View style={[style, styles.tile]}>
+      <Text style={styles.tiletxt}>{data}</Text>
+      <Pressable onPress={() => deleteTask(data)} style={styles.delete}>
+        <AntDesign name="close" size={24} />
+      </Pressable>
+    </Animated.View>
+  );
+}
+
+function E({ route, navigation }: StackScreenProps<'E'>) {
+  /** save tasks to async Storage to retrive later */
+  async function saveTaskList() {
+    try {
+      await AsyncStorage.setItem('@saved_tasks', JSON.stringify(newOrder));
+    } catch (err) {
+      console.warn(err);
+    }
+  }
+  const OGTasks = route.params.tasks;
+  const [data, setData] = useState<string[]>(route.params.tasks.map((item) => item.descr));
+  const [newOrder, setNewOrder] = useState<TaskProp[]>(OGTasks);
+
+  const renderRow = useCallback(({ data, active }) => {
+    return <Row active={active} data={data} deleteTask={deleteTask} />;
+  }, []);
+
+  /** delete a task & save neworder */
+  const deleteTask = (taskName: string) => {
+    const tmp = data.filter((elem) => elem !== taskName);
+    setData((prevData) => prevData.filter((elem) => elem !== taskName));
+    setNewOrder((prevOrder) => tmp.map((elem) => prevOrder.filter((p) => p.descr === elem)).flat());
+  };
+
+  /** save newOrder of task for later */
+  const onReleaseRow = (key: string, currentOrder: string[]) => {
+    setNewOrder(currentOrder.map((item) => OGTasks.filter((entry) => parseInt(item) === entry.id)).flat());
+  };
+  const handleGoOn = () => {
+    saveTaskList();
+    navigation.navigate('N');
+  };
+
+  return (
+    <>
+      <Title back color={PURPLE} text="Situationskontrolle" />
+      <ScrollView contentContainerStyle={styles.container}>
+        <View>
+          <Text style={styles.para}>
+            Jetzt geht es darum die Aufgaben nach der Wichtigkeit zu sortieren. Du kannst die Aufgaben einfach in die
+            gewünscht Reihenfolge verschieben.
+          </Text>
+        </View>
+        <View>
+          <Text style={styles.heading}>
+            <Text style={styles.accent}>E</Text>ntscheidungen treffen:
+          </Text>
+          <Text style={styles.description}> a) Das Wichtigste zuerst </Text>
+          <Text style={styles.description}> b) Es ist okay, wenn du nicht alles schaffst!</Text>
+        </View>
+
+        <SortableList data={data} onReleaseRow={onReleaseRow} renderRow={renderRow} style={styles.sortContainer} />
```

Comment:
Hier ist es für mich zumindest nicht möglich die Anordnung der Aufgaben zu ändern, liegt vielleicht am äußeren ScrollView? Hat bei mir auf Android und iOS leider nicht funktioniert.

Reply:
Muss anscheinend komplett überarbeitet werden, bei android funktioniert es jetzt bei mir auch gar nicht mehr...

Reply:
Hmm können wir wie gesagt gerne in einem Separaten Issue machen, da es auch nicht so ein kritisches Feature ist. Mir sit bei der aktuellen Version noch aufgefallen, dass wenn man beim anordnen der Prioritäten das oberste oder das unterste nach unten verschiebt, dann wird die Postion im Hintergrund weiter verschoben, obwohl sich die Position in der Liste nicht ändert. Dadurch muss man dann die restlichen Aufgaben genau so oft verschieben, damit sich deren Position in der Liste ändert. Falls das zu verwirrend war kann ich gerne kurz ein video davon machen :)